index.js

```diff
@@ -1560,9 +1560,11 @@ Buffer.prototype.fill = function fill (val, start, end, encoding) {
 // HELPER FUNCTIONS
 // ================
 
-var INVALID_BASE64_RE = /[^+/0-9A-Za-z-_]/g
+var INVALID_BASE64_RE = /[^+/0-9A-Za-z-_=]/g
```

Comment:
I don't think this line needs to be changed. In the new line you added below, you're already finding the first `=` and discarding it and everything that follows. There should be no `=` left in the string to remove.